esp/smc/SMCLib/LogicFileWrapper.cpp

```diff
@@ -107,7 +108,7 @@ IDistributedFile* lookupLogicalName(IEspContext& context, const char* logicalNam
         userDesc->set(userID, context.queryPassword(), context.querySignature());
     }
 
-    Owned<IDistributedFile> df = queryDistributedFileDirectory().lookup(logicalName, userDesc, accessMode,
+    Owned<IDistributedFile> df = wsdfs::lookup(logicalName, userDesc, accessMode,
```

Comment:
May not be related to this issue. I changed it because of your comment in https://github.com/hpcc-systems/HPCC-Platform/pull/18013#discussion_r1394061066, and because it is called by CFileSprayEx::onCopy through the getNodeGroupFromLFN(). If you want, I can change it back.